.github/workflows/main.yml

```diff
@@ -24,9 +24,9 @@ jobs:
       - name: Set env
         id: vars
         run: |
-          echo ::set-output name=tag::${GITHUB_REF#refs/*/}
-          echo ::set-output name=short_commit::$(echo $GITHUB_SHA | cut -c1-7)
-          echo ::set-output name=date::$(date -u +"%Y-%m-%dT%H:%M:%SZ")
+          echo name=${GITHUB_REF#refs/*/} >> $GITHUB_OUTPUT
+          echo name=$(echo $GITHUB_SHA | cut -c1-7) >> $GITHUB_OUTPUT
+          echo name=$(date -u +"%Y-%m-%dT%H:%M:%SZ") >> $GITHUB_OUTPUT
```

Comment:
In the original example, the text is surrounded by quotes. I am not sure if it's important or not, but yaml is sometimes confused with special characters. I would consider adding the double quotes just to be sure...

e.g.
```
echo "tag=${GITHUB_REF#refs/*/}" >> $GITHUB_OUTPUT
```